src/sagas/authentication.js

```diff
@@ -18,15 +18,15 @@ function* loginGoogleRequest() {
             window._UI_STORE_.dispatch( setCurrentUser({permissions: {}}) );
     });
     console.log('You have been logged in');
-    
+
     yield;
 }
 
 function* logoutUserRequest() {
     window._FIREBASE_.auth().signOut();
     window._UI_STORE_.dispatch( setCurrentUser({permissions: {}}) );
     console.log('You have been logged out');
-    console.log('TODO: refresh the page')
+    window.location.reload();
```

Comment:
Instead of doing a full app refresh, dispatching an action that sets the current user to an empty object will have the same effect with minimal side effects. Maybe `setCurrentUser`? Not sure if it would handle an empty user object or not.